include/config.hpp

```diff
@@ -13,6 +13,18 @@
 /*
  * Serial config
  */
+// uncomment the below line to use a seperate UART port for radio communication
+// this frees UART0 and the USB port for programming
+#define USE_UART2_FOR_RADIO  // UART2 is readily useable, UART1 requires more configuration as it shares pins with SPI
```

Comment:
Is there a reason for this? We can simply move to UART2.

Reply:
Modified. Now using UART2 for radio by default.